stable/kong/values.yaml

```diff
@@ -438,3 +438,9 @@ serviceMonitor:
   # interval: 10s
   # Specifies namespace, where ServiceMonitor should be installed
   # namespace: monitoring
+  # The prometheus-operator selects scrape targets via a label on the ServiceMonitor,
+  # this label can be custom per the operators configuration.
+  prometheusOperatorServiceMonitorSelector:
+    enabled: false
+    matchLabels:
+      prometheus: somelabel
```

Comment:
This looks very weird.
Why is this additional level of `prometheusOperatorServiceMonitorSelector` required?

I'd propose that we do something simpler like:

```yaml
serviceMonitor:
  enabled: false
  labels:
    foo: bar
```

Also, enabled for the labels  should not be necessary for labels.
You can simply check if any labels are set and inject them:
https://github.com/helm/charts/blob/master/stable/kong/templates/deployment.yaml#L31

Reply:
Good point =)....updating...

Reply:
@hbagdi updated.  This looks way better than my long variable name =)